catlover.client/src/components/cats/catsComponent.jsx

```diff
@@ -0,0 +1,98 @@
+import React, { Component } from "react";
+import CatComponent from "./catComponent";
+import LoadingComponent from "../loading/loadingComponent";
+import ErrorComponent from "../error/errorComponent";
+import PageTitleComponent from "../pageTitle/pageTitleComponent";
+import PaginationComponent from "../pagination/paginationComponent";
+
+class CatsComponent extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      currentPage: 1,
+      catsPerPage: 10,
+      error: null,
+      isLoaded: false,
+      message: String,
+      items: [],
+    };
+  }
+
+  componentDidMount() {
+    fetch("http://localhost:10000/cats")
```

Comment:
How come you didn't add API_URL similar to `catComponent` above?

Reply:
You are right. This is not good approach. The correct way is to use one time the API_URL in a generic service file and this API_URL in my opinion must be saved in a data table "tblConfig" in case we want to change it dynamically some other day. Also all other string controllers/endpoints in the same table (cats, breeds etc.)